--- a/src/misc.cpp
+++ b/src/misc.cpp
@@ -43,7 +43,7 @@
 // Version number or dev.
-constexpr std::string_view version = "dev";
+constexpr std::string_view version = "2026-01-02";
 
 // Our fancy logging facility. The trick here is to replace cin.rdbuf() and
 // cout.rdbuf() with two Tie objects that tie cin and cout to a file stream. We
 // can toggle the logging of std::cout and std::cin at runtime whilst preserving
